components/feature/pwa/src/test/java/mozilla/components/feature/pwa/ext/ActivityKtTest.kt

```diff
@@ -66,13 +72,18 @@ class ActivityKtTest {
 
     @Test
     fun `WHEN reportFullyDrawnSafe is called THEN reportFullyDrawn is called`() {
-        activity.reportFullyDrawnSafe()
+        activity.reportFullyDrawnSafe(logger)
         verify(activity).reportFullyDrawn()
     }
 
     @Test
-    fun `GIVEN reportFullyDrawn throws a SecurityException wHEN reportFullyDrawnSafe is called THEN the exception is caught`() {
-        `when`(activity.reportFullyDrawn()).thenThrow(SecurityException())
-        activity.reportFullyDrawnSafe() // If an exception is thrown, this test will fail.
+    fun `GIVEN reportFullyDrawn throws a SecurityException WHEN reportFullyDrawnSafe is called THEN the exception is caught and a log statement with fully drawn is logged`() {
+        val expectedSecurityException = SecurityException()
+        `when`(activity.reportFullyDrawn()).thenThrow(expectedSecurityException)
+        activity.reportFullyDrawnSafe(logger) // If an exception is thrown, this test will fail.
+
+        val msgArg = ArgumentCaptor.forClass(String::class.java)
```

Comment:
There's also an inline `argumentCaptor` function for this since the built-in Mocktio version sometimes fails. I'm happy to just use `any` as well.